taxi/views.py

```diff
@@ -59,6 +81,22 @@ class CarListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
     queryset = Car.objects.select_related("manufacturer")
 
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super().get_context_data(**kwargs)
+        model = self.request.GET.get("model", "")
+        context["search_form"] = CarSearchForm(
+            initial={"model": model}
+        )
+        return context
+
+    def get_queryset(self):
+        form = CarSearchForm(self.request.GET)
+        if form.is_valid():
+            return self.queryset.filter(
```

Comment:
it's not recommended to use self.queryset because if you inherit from another view, that has get_queryset() method overrided, but not queryset attribute, then you wan't reuse this logic. fix everywhere